lstchain/mc/plot_utils.py

```diff
@@ -135,24 +136,48 @@ def format_axes_sensitivity(ax):
     ax.set_ylabel(r'E$^2$ $\frac{\mathrm{dN}}{\mathrm{dE}}$ [TeV cm$^{-2}$ s$^{-1}$]')
     ax.grid(ls='--', alpha = .5)
 
-def plot_MAGIC_sensitivity(ax):
+def plot_LST_preliminary_sensitivity(ax, **kwargs):
+    """
+    Plot LST preliminary sensitivity for comparison with the reached one
+
+    Parameters
+    --------
+    ax:    `matplotlib.pyplot.axis`
+
+    Returns
+    --------
+    ax:    `matplotlib.pyplot.axis`
+
+    """
+    s = np.loadtxt('../spectra/data/sensitivity.txt', skiprows = 1)
```

Comment:
this file is missing

Reply:
Yes, I think I will remove this function, since I don't think is somenthing very general. I will take a look at the rest of the codecov issues asap.